### ISSUE TITLE
Top and left of pie chart always squared cut off.

### ISSUE THREAD

**Issue by ggaabe** [tomauty/react-native-chart#82]
Hola,

I said I would make another issue and now I am!

So the top of the pie chart is always slightly cut off. Even if I add to the marginTop to push it away from other things, or make the pie chart twice as large, it remains just slightly squared down, like a razor scooter wheel that someone hit the brakes on for too long while skidding:
![screen shot 2016-06-17 at 3 27 42 pm](https://cloud.githubusercontent.com/assets/5441185/16163697/60840f7c-34a0-11e6-8103-3f3f226e38d8.png)
![screen shot 2016-06-17 at 3 27 30 pm](https://cloud.githubusercontent.com/assets/5441185/16163700/66198e58-34a0-11e6-9ed0-d796af2bf789.png)

I think the grid, whether visible or not, possibly slightly affecting it.
Note: This is much more noticeable in iOS simulation than it is when compiled on a physical device. It's not really noticeable on a physical device unless you really have a compulsion for perfection.

**Comment by tomauty** [comment]
Ah yeah, I've noticed this too. Not sure what the deal is. My guess was that the radius may be too large for the container, but then I would think all sides would be affected.

**Comment by ggaabe** [comment]
I think the center of the circle isn't in the dead center of the available space. If you look at the second picture, there's just a little bit of space between the bottom of the circle and the x axis, but on the left side, there is absolutely no space between the pie and the y axis. The circle touches it, and the rest is cut off. If the center is shifted slightly rightward and downward, it should be good.

**Comment by tomauty** [comment]
I wonder if it has to do with this layout initialization code in `Chart.js`

``` javascript
    _onContainerLayout = (e : Object) => this.setState({
        containerHeight: Math.ceil(e.nativeEvent.layout.height) + 1,
        containerWidth: Math.ceil(e.nativeEvent.layout.width),
    });
```

Maybe the ceil and +1 are shifting over somehow?

**Comment by ggaabe** [comment]
Yeah, ceil lacks precision because it'll always bump up to the nearest
integer. Maybe you should add 1 and then round on the height?
Like this:
`containerHeight: Math.round(e.nativeEvent.layout.height + 1)`

And since it's a little too far over to the left, maybe we could add 1 to the containerWidth as well?

**Comment by rgoldiez** [comment]
Is this a case for using the PixelRatio API?  There's a method for getting the rounding right for different screen DPIs.

**Comment by tomauty** [comment]
Ah right, that's probably what I'm looking for in this case. I'll experiment. Feel free to do the same :)

**Comment by tomauty** [comment]
I tried out `PixelRatio`, but it seems it doesn't really fit our use case because we're looking for _smaller_ numbers to round. The lowest `PixelRatio.get()` response is 1.

**Comment by tomauty** [comment]
As a generic FYI — I am no longer able to maintain this library. I recommend checking out [victory-native](https://github.com/FormidableLabs/victory-native) as it's much more maintained.
